lib/src/mount.rs

```diff
@@ -57,3 +57,24 @@ pub(crate) fn mount(dev: &str, target: &Utf8Path) -> Result<()> {
         [dev, target.as_str()],
     )
 }
+
+/// If the fsid of the passed path matches the fsid of the same path rooted
+/// at /proc/1/root, it is assumed that these are indeed the same mounted
+/// filesystem between container and host.
+/// Path *should not* include a leading '/'. It will be stripped.
```

Comment:
But we do pass a leading `/` in the single caller :smile: 

How about let's just say it should be absolute and rely on the `strip_prefix`  below.

Reply:
Good point.